frontend/src/screens/studyBanner.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useCallback } from 'react';
+import { Box, Title, Text, Anchor, Group, useMantineTheme } from '@mantine/core';
+import { IconArrowRight } from '@tabler/icons-react';
+import { useNavigate } from 'react-router-dom';
+import { colors } from '@theme';
+import { useSearchStudies, useParticipantStudies } from './learner/studies';
+
+interface ParticipantStudy {
+    id: string;
+    completedCount?: number;
+    learningPath?: {
+        completed?: boolean;
+    };
+    totalPoints?: number;
+    stages?: { durationMinutes: number }[];
+    completedAt?: string;
+}
+
+const StudyBanner: React.FC = () => {
+    const { filteredStudies } = useSearchStudies();
+    const { studies = [] } = useParticipantStudies() as { studies: ParticipantStudy[] };
+    const navigate = useNavigate();
+    const theme = useMantineTheme();
+
+    const totalCompletedCount = studies.reduce(
```

Comment:
Do you want these to run on every render or should these be in some sort of useEffect to only run when studies change?